build/ci-linux.sh

```diff
@@ -0,0 +1,28 @@
+IMAGE="$1"	
+ARCH="$2"	
+DOCKER_USER="$3"	
+DOCKER_PASS="$4"
+APPVEYOR_REPO_BRANCH="$5"
+APPVEYOR_PULL_REQUEST_NUMBER="$6"
+
+if [ "${APPVEYOR_PULL_REQUEST_NUMBER}" ]; then
+  tag="pr${APPVEYOR_PULL_REQUEST_NUMBER}-$IMAGE-$ARCH"
+  manifest="pr${APPVEYOR_PULL_REQUEST_NUMBER}"
+else
+  tag="${APPVEYOR_REPO_BRANCH}-$IMAGE-$ARCH"
+  manifest="${APPVEYOR_REPO_BRANCH}"
+fi
+
+docker build -t "ssbkang/portainer:$tag" -f build/linux/Dockerfile .
```

Comment:
We probably want to update any `ssbkang` occurence to `portainer`.

Reply:
Yep, I will make the change